flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java

```diff
@@ -487,6 +489,21 @@ public void visit(CheckExpressionNode checkExpressionNode) {
         nodeBuilder.codedata().lineRange(checkExpressionNode);
     }
 
+    @Override
+    public void visit(MappingConstructorExpressionNode mappingCtrExprNode) {
```

Comment:
Shall we do this for ListConstructor too?

Reply:
Added